drakrun/drakrun/main.py

```diff
@@ -287,12 +287,19 @@ def crop_dumps(self, dirpath, target_zip):
         max_total_size = 300 * 1024 * 1024  # 300 MB
         current_size = 0
 
+        dumps_metadata = []
         for _, path, size in sorted(entries):
             current_size += size
 
             if current_size <= max_total_size:
                 # Store files under dumps/
-                zipf.write(path, os.path.join("dumps", os.path.basename(path)))
+                file_basename = os.path.basename(path)
+                if re.match(r"^[a-f0-9]{4,16}_[a-f0-9]{16}$", file_basename):
+                    # If file is memory dump then append metadata that can be
+                    # later attached as payload when creating an `analysis` task.
+                    dump_base = self._get_base_from_drakrun_dump(file_basename)
+                    dumps_metadata.append({"filename": os.path.join("dumps", file_basename), "base_address": dump_base})
```

Comment:
Would it be possible to use `hex(dump_base)` ? I know Python supports arbitrary size integers when serializing JSON, however this may cause some issues when interoperating with other JSON libraries.
Also, hex strings would be much more readable